showcase/tests/integration/components/hds/copy/button/index-test.js

```diff
@@ -46,6 +46,22 @@ module('Integration | Component | hds/copy/button/index', function (hooks) {
     assert.true(this.success);
   });
 
+  test('it should copy an empty string value passed in', async function (assert) {
```

Comment:
[thought] should these just live directly in the tests for the underlying modifier? in this way both `Copy::Button` and `Copy::Snippet` are covered